tests/integration/targets/info_test/run.yml

```diff
@@ -1,19 +1,37 @@
-- hosts: localhost
-  gather_facts: no
-  collections:
-    - community.general
-  vars_files:
-    - vars.yml
+- name: Run Info Test Tasks
+  hosts: localhost
+  gather_facts: false
 
   tasks:
+
+    - name: Import eco-vcenter credentials
+      ansible.builtin.include_vars:
+        file: ../../../../scripts/integration_config.yml
+      tags: eco-vcenter-ci
+
+    - name: Import vars for testing on eco-vcenter env
+      ansible.builtin.include_vars:
+        file: ../vars/main.yml
```

Comment:
you can skip this task since variables in vars/main.yml are already included. They should be overwritten by variables in vars.yml when the tag is `integration-ci`

Reply:
tnx! done :D